docs/source/release/templates/framework.rst

```diff
@@ -0,0 +1,61 @@
+=================
+Framework Changes
+=================
+
+.. contents:: Table of Contents
+   :local:
+
+Algorithms
+----------
+
+New features
+############
+.. amalgamate:: Framework/Algorithms/New_features
+
+Bugfixes
+############
+.. amalgamate:: Framework/Algorithms/Bugfixes
+
+Fit Functions
+-------------
+
+New features
+############
+.. amalgamate:: Framework/Fit_Functions/New_features
+
+Bugfixes
+############
+.. amalgamate:: Framework/Fit_Functions/Bugfixes
+
+
+Data Objects
+------------
+
+New features
+############
+.. amalgamate:: Framework/Data_Objects/New_features
+
+Bugfixes
+############
+.. amalgamate:: Framework/Data_Objects/Bugfixes
+
+
+Python
+------
+
+New features
+############
+.. amalgamate:: Framework/Python/New_features
+
+Bugfixes
+############
+.. amalgamate:: Framework/Python/Bugfixes
+
+
+Installation
```

Comment:
Installation is not mentioned in the Release Note Guide

Reply:
@martyngigg I'm not sure this is a heading we really use. Should we remove it from the template or leave it?

Reply:
Ok so I've thought about this and I'm just going to remove Installation from the templates. We can always add it when we want to use it but I just don't feel its needed to be there by default.

Reply:
Sorry for the delay in reply!
We generally don't affect the installation much so as a general rule it probably doesn't deserve being there as standard so I agree removing from the template makes sense.